projects/storefrontlib/src/shared/components/anonymous-consents-dialog/anonymous-consent-dialog.component.ts

```diff
@@ -39,15 +39,15 @@ export class AnonymousConsentDialogComponent implements OnInit, OnDestroy {
   focusConfig: FocusConfig = {
     trap: true,
     block: true,
-    autofocus: true,
+    autofocus: 'input[type="checkbox"]',
```

Comment:
owh, are we in agreement now? :D

Reply:
Always, except when we are not 😉. But yes I agree on this now.